### DISCUSSION TITLE
Question about diff of personal and pro plans

### DISCUSSION THREAD

**Discussion by jamesarm97** [community/community#34486] [Other Feature Feedback, Questions, & Ideas]
I have been paying for the Github Pro plan for years and use it to store about 49 private repositories of code I work on. I have looked at downgrading to the free plan because the Pro plan shows nothing in the "numbers" used (space, actions, etc). Every time I go to downgrade I get scared I will loose something. Everything is plain vanilla, just private repositories no teams or other editing members. Will I be ok downgrading? If something is lost that is in the pro plan, is there a grace period to upgrade and get it back or am I ok with just having the free plan with unlimited repositories? I did the download data archive and it was 1Gb compressed.

So, Github Pro -> personal Free?

![Screen Shot 2022-09-28 at 2 20 31 PM](https://user-images.githubusercontent.com/929192/192859123-f050e734-eb60-46c2-8612-892d475097a0.png)

**Comment by radoveneanuvlad** [comment]
I think github pro plan is a great ideea for someone that wants the full potential of github. do you agree?
